src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -280,12 +281,23 @@ export default class StandardRouteWaypointModel extends BaseModel {
      * Find the matching fix from the `FixCollection` and clone its `StaticPositionModel` this `_positionModel`
      *
      * @for StandardRouteWaypointModel
-     * @method _clonePoisitonFromFix
+     * @method clonePositionFromFix
      * @param fixCollection {FixCollection}
      * @private
      */
-    clonePoisitonFromFix() {
-        const fixModel = FixCollection.findFixByName(this.name);
+    clonePositionFromFix() {
+        if (this.name[0] === VECTOR_WAYPOINT_PREFIX) {
+            return;
+        }
+
+        let name = this.name;
+
+        if (this.name[0] === HOLD_WAYPOINT_PREFIX
+            || this.name[0] === FLY_OVER_WAYPOINT_PREFIX) {
```

Comment:
the second one
```javascript
// instead of 
if (this.name[0] === HOLD_WAYPOINT_PREFIX
    || this.name[0] === FLY_OVER_WAYPOINT_PREFIX) { }

// preferred
if (this.name[0] === HOLD_WAYPOINT_PREFIX || 
    this.name[0] === FLY_OVER_WAYPOINT_PREFIX) { }
```